python/ray/experimental/state.py

```diff
@@ -558,19 +573,35 @@ def micros_rel(ts):
             delta_info["function_name"] = info["function_name"]
             delta_info["worker_id"] = info["worker_id"]
             worker = workers[info["worker_id"]]
+            task_t_info = task_table[task_id]
+            task_spec = task_table[task_id]["TaskSpec"]
+            task_spec["Args"] = [oid.hex() if type(oid) is not int else oid
+                                 for oid in task_t_info["TaskSpec"]["Args"]]
+            task_spec["ReturnObjectIDs"] = [oid.hex() for oid in
+                                            (task_t_info["TaskSpec"]
+                                             ["ReturnObjectIDs"])]
+            task_spec["LocalSchedulerID"] = task_t_info["LocalSchedulerID"]
+
+            total_info = dict(delta_info, **task_spec)
+
+            parent_info = task_info.get(
+                            task_table[task_id]["TaskSpec"]["ParentTaskID"])
+            worker = workers[info["worker_id"]]
+
             if breakdowns:
                 if "get_arguments_end" in info:
                     get_args_trace = {
                         "cat": "get_arguments",
                         "pid": "Node " + str(worker["node_ip_address"]),
                         "tid": info["worker_id"],
-                        "id": str(worker),
+                        "id": str(task_id),
```

Comment:
task_id is already a string, so the `str` should not be necessary. This applies in several more places.